scripts/wsrep_sst_common.sh

```diff
@@ -648,31 +556,29 @@ function write_wsrep_state_file()
 #   (1) Receiving an SST from a 5.7 donor
 #       The basic flow is:
 #           1. start up a MySQL server (#1) (creates the SST user)
+#               The upgrade will be performed here (if needed).
 #           2. shut down MySQL server
 #               Because the system tables use MyISAM (5.7 donor), the
 #               account mgmt commands (like CREATE USER, FLUSH PRIVILEGES)
 #               will not work. So the only way to use this new user is
 #               to restart the server.  Note that the SST user is not
 #               removed on shutdown, because the 'DROP USER' call will fail.
-#           3. start up MySQL Server (#2)
-#           4. run mysql_upgrade with the SST user
-#           5. shut down the MySQL server
+
 #               The loading of any async replication info will have failed
 #               because the tables are MyISAM. So we need to restart (again)
 #               so that we can remove the async replication info. Note that
 #               this time, the SST user will be removed on shutdown.
-#           6. start up MySQL Server (#3) (creates the SST user)
-#           7. run 'RESET SLAVE ALL' (if needed)
-#           8. shutdown the MySQL Server
+#           3. start up MySQL Server (#3) (creates the SST user)
+#           4. run 'RESET SLAVE ALL' (if needed)
+#           5. shutdown the MySQL Server
 #
-#   TODO: Steps (5) and (6) are needed due to 8.0.13 not restoring the
+#   TODO: Steps (2) and (3) are needed due to 8.0.13 not restoring the
```

Comment:
8.0.18

Reply:
ok (i actually removed the entire comment section since it's out of date)